todoman/ui.py

```diff
@@ -28,15 +28,18 @@ def __init__(self, todo, lists, formatter):
         """
         :param model.Todo todo: The todo object which will be edited.
         """
-
+        
+        self.current_list = todo.list
         self.todo = todo
         self.lists = lists
         self.formatter = formatter
         self.saved = EditState.none
         self._loop = None
 
         self._msg_text = urwid.Text('')
-
+        urwid.RadioButton(self.lists, "label", state='first True',
```

Comment:
* You are creating a RadioButton and discard it. Instead, you need to add it to the `pile_items` list or `items`, depending on which position you want the list selector to be. In *which position of the list* you want to add the `RadioButton` also depends on where you want the list selector to be.
* You will need multiple RadioButtons, one for each list to select.
* The first argument has to be a list to which the RadioButton will append itself to. `self.lists` cannot be that list. It should be `items` or `pile_items`, but since that list is only created further down, you will need to move the entire block down.
* `on_state_change` has to be a function that sets `self.current_list` to the list the RadioButton stands for. Something like this:

      def change_current_list(new_list):
          self.current_list = new_list

      for _list in self.lists:
          RadioButton(..., on_state_change=change_current_list, user_data=_list)